Library/Homebrew/tap.rb

```diff
@@ -434,7 +434,7 @@ def install(quiet: false, clone_target: nil,
       return
     elsif (core_tap? || core_cask_tap?) && !Homebrew::EnvConfig.no_install_from_api? && !force
       odie "Tapping #{name} is no longer typically necessary.\n" \
-           "Add #{Formatter.option("--force")} if you are sure you need it done."
+           "Add #{Formatter.option("--force")} if you are sure you need it for local development."
```

Comment:
@apainintheneck `for contributing to Homebrew` might be better here.

`local development` does not imply `local development of Homebrew` to me.

Similarly, could consider not requiring `--force` if `devcmdrun` is true as that at least somewhat implies that they are doing local development.

Reply:
I like that wording better. I'll update it.

I'm fine with the idea of not requiring `--force` but I'd like to get other people's opinions on that. It might be controversial.

Reply:
For documentation: The main reason to keep the `--force` around here is to prevent copy and paste errors with old command line snippets littered around the internet.